cmd/config-current.go

```diff
@@ -27,44 +27,35 @@ import (
 	"github.com/tidwall/gjson"
 )
 
+// Steps to move from version N to version N+1
+// 1. Add new struct serverConfigVN+1 in config-versions.go
+// 2. Set configCurrentVersion to "N+1"
+// 3. Set serverConfigCurrent to serverConfigVN+1
+// 4. Add new migration function (ex. func migrateVNToVN+1()) in config-migrate.go
+// 5. Call migrateVNToVN+1() from migrateConfig() in config-migrate.go
+// 6. Make changes in config-current_test.go for any test change
+
 // Config version
-const v20 = "20"
+const configCurrentVersion = "20"
+
+type serverConfigCurrent = serverConfigV20
```

Comment:
variable name ``` serverConfigCurrent ``` can be changed to either ``` serverConfig ``` or ``` currentServerConfig ```

Reply:
yes, good suggestion, will change it to serverConfig